packages/teams-js/src/public/app.ts

```diff
@@ -354,6 +354,11 @@ export namespace app {
      * ID for the current visible app which is different for across cached sessions. Used for correlating telemetry data.
      */
     appLaunchId?: string;
+
+    /**
+     * App id that is used by Hosts to distinguish between different apps sideloaded or in store
+     */
+    appId: string;
```

Comment:
This will always be populated as long as there is an app def sent back; I rewrote the code in hubSDK so that hubs no longer need to pass the value and instead it is taken from app definition

Reply:
What about hosts that aren't yet using the updated host-sdk? I believe in that case this would not be set, right? That's generally why we mark things like this optional, to handle backwards compatibility